qualtran/bloqs/mod_arithmetic/mod_addition.py

```diff
@@ -88,8 +89,8 @@ def on_classical_vals(
         return {'x': x, 'y': (x + y) % self.mod}
 
     def build_composite_bloq(self, bb: 'BloqBuilder', x: Soquet, y: Soquet) -> Dict[str, 'SoquetT']:
-        if is_symbolic(self.bitsize):
-            raise NotImplementedError(f'symbolic decomposition is not supported for {self}')
+        if isinstance(self.bitsize, sympy.Expr):
```

Comment:
why the change from `is_symbolic` for `isinstance`?

Reply:
I thought that was the reason some tests would fail for symbolic decomposition - now realizing it was the DecomposeTypeError vs NotImplementedError.